packages/util-dynamodb/src/marshall-integ.spec.ts

```diff
@@ -0,0 +1,143 @@
+import { marshallInput } from "../../../lib/lib-dynamodb/src/commands/utils";
+import { marshall } from "./marshall";
+import { convertToAttr } from "./convertToAttr";
```

Comment:
The marshal unit tests need go to `marshall.spec.ts`

Reply:
If you plan to write integration tests, please using `.integ.spec.ts`

Reply:
I've opted not to use integration tests, or name anything `.integ.spec.ts`. It seems there are no tests of that pattern in our libs and packages and the jest config is incomplete for supporting that. I created a `marshallInput.spec.ts` instead. 

As for the additional `marshall` unit tests, I tried to put them in the existing `marshall.spec.ts` but couldn't find a working jest setup to both mock and unmock `convertToAttr` in the same file. It was expedient to put it in a second file, if that's ok? I tried the jest docs for mocking and unmocking in the same file, but in the short amount of time I spent I couldn't get it working. The setup is also verbose compared to having a second file.

Reply:
> It seems there are no tests of that pattern in our libs

We do. But we use it only for actual integration tests https://github.com/aws/aws-sdk-js-v3/blob/main/clients/client-transcribe-streaming/test/index.integ.spec.ts

> jest config is incomplete for supporting that.


Our default jest config will run based on configurations defined in `jest.config.js` in each related packages.
 https://github.com/aws/aws-sdk-js-v3/blob/99235b45be62706d77ce93f22f8cac736cd9ce1b/jest.config.js#L1-L7
 
The default jest config in util-dynamodb extends from base which skips `*.integ.ts`
 https://github.com/aws/aws-sdk-js-v3/blob/99235b45be62706d77ce93f22f8cac736cd9ce1b/jest.config.base.js#L5

Reply:
> I created a marshallInput.spec.ts instead.

This should work.